### ISSUE TITLE
Maintenance of this crate/repository is in an awkward spot

### ISSUE THREAD

**Issue by nagisa** [standard-ai/tracing-gstreamer#16]
I’m effectively the only person who maintains this project, and I’m the only person who has the rights to publish new versions of this crate.

However, as part of a recentish permissions cleanup, I was removed as an administrator of this repository. Even though I’m no longer an employee at Standard for a long time at this point, this action seems to go against the spirit of the `standard-ai` organization which was meant to host OSS code. OSS is meant to transcend organizational boundaries in a sense – if it is required that only employees of an organization can be contributors or maintainers, then it isn’t quite OSS anymore, is it?

But I digress, my intent is to resolve practical issues that arise from the maintainer not having administrative access to the repository. In particular I no longer have an ability to setup things such as GHA CI, add additional contributors/maintainers or e.g. set up new features such as merge queues. This incapacitates me as a maintainer. I guess I could still fork the repository and adjust the links, but that doesn’t sound like a great way forward.

If Standard is not interested in maintaining this crate, and would rather not have any former employees with administrative permissions in the `standard-ai` project, I would suggest that this crate is moved to myself proper.

If, however, Standard does intend to step up to maintain this code instead, I’m happy to step down and replace myself with some `@standard-ai/*` team as a crate owner/publisher.

Random pings to who may still be able to help here: cc @maroux @rnarubin

**Comment by maroux** [comment]
@nagisa updated your permissions to admin. Hope that helps.

**Comment by sdroege** [comment]
@nagisa It could also be an option to maintain this as part of GStreamer. I don't have a strong opinion about that but @thiblahute mentioned that the other day.

@gdesmott probably also has opinions on that.

**Comment by gdesmott** [comment]
> It could also be an option to maintain this as part of GStreamer.

That would make sense, especially if we are multiple GStreamer developers using this crate already.

**Comment by nagisa** [comment]
I wouldn’t be against handing this code off to the GStreamer project proper, but I think it would be a good idea to at least get a nod of approval from Standard who has funded development of this code in the first place.

Unfortunately I no longer have any remaining contacts within the company and my best bet is to ping Maru as I did above.

---

@maroux would it be alright with you to ask around if it'd be fine to move this project into GStreamer proper? I can handle the rest so long as I have an approval.

**Comment by sdroege** [comment]
If we move it, we'll also make sure to give you useful permissions FWIW :) I guess moving would at least give more visibility.

**Comment by rnarubin** [comment]
> I wouldn’t be against handing this code off to the GStreamer project proper, but I think it would be a good idea to at least get a nod of approval from Standard who has funded development of this code in the first place.
> 
> Unfortunately I no longer have any remaining contacts within the company and my best bet is to ping Maru as I did above.
> 
> @maroux would it be alright with you to ask around if it'd be fine to move this project into GStreamer proper? I can handle the rest so long as I have an approval.

I'll let Maru weigh in about process, I'd consider it a great success to have this adopted by the GStreamer project cc @blogle

**Comment by blogle** [comment]
I am aligned with @rnarubin, I think it would be great if the gstreamer team wants to take ownership of this project going forward. We are still utilizing this code to a degree across projects internally, but have not dedicated too many cycles to maintenance or improvements. We are more dependent on some custom tracing elements that export various metrics (latency, throughput, queue levels, etc) over opentelemetry.

**Comment by maroux** [comment]
@sdroege @nagisa 👍 on this move from [Sc] side.

**Comment by sdroege** [comment]
Great, thanks :) @thiblahute do you want to take care of this?

**Comment by thiblahute** [comment]
Great news indeed, thanks. I am happy to work on it, but won't be able to do it in the coming weeks I am afraid.

**Comment by thiblahute** [comment]
This was now merged upstream: https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/-/merge_requests/1900

Thanks a lot.

**Comment by nagisa** [comment]
@thiblahute what do you want to do with the current `tracing-gstreamer` crate? My thinkin is that I redirect users to `gstreamer-tracing` in README, but leave crate code intact (i.e. don't reexport everything from `tracing-gstreamer` and have the crate emit a warning, but I could do that...?)

**Comment by thiblahute** [comment]
Yeah, changing the README to point to gstreamer-tracing and then the crate emit a warning sounds good to me.
